AS_920_923_JP_TTN_1.yml

```diff
@@ -0,0 +1,88 @@
+band-id: AS_920_923_JP_TTN_1
```

Comment:
I don't think so.
AS_923 contains several countries, AS_920_923_JP_TTN_1 is made for Japan.
FYI1) AS923_3, AS923_4 groups are not valid for Japan. Because the default channels these groups have are not compliant to ARIB STD-T108.
FYI2) I think AS923_1 is better for Japan. Because the default channels of AS923_1 are already de-fact standard.

Reply:
Right, so this should be `AS923` then. Note that this field is the _band_, not the identifier of this frequency plan.